orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/LevelOfConfidence.java

```diff
@@ -6,6 +6,8 @@
 
 public enum LevelOfConfidence {
     NONE("P0", true),
+    HMRC1("PCl200", true),
+    HMRC2("PCl250", true),
```

Comment:
Nitpick, but
```suggestion
    HMRC200("PCl200", true),
    HMRC250("PCl250", true),
```
I think HMRC have other levels we may at some point support

Reply:
Yeah agreed